localstack/services/cloudformation/models/awslambda.py

```diff
@@ -121,16 +129,20 @@ def get_environment_params(params, **kwargs):
             "create": {
                 "function": "create_function",
                 "parameters": {
-                    "FunctionName": "FunctionName",
-                    "Runtime": "Runtime",
-                    "Role": "Role",
-                    "Handler": "Handler",
+                    "Architectures": "Architectures",
                     "Code": LambdaFunction.get_lambda_code_param,
                     "Description": "Description",
                     "Environment": get_environment_params,
-                    "Timeout": "Timeout",
+                    "FunctionName": "FunctionName",
+                    "Handler": "Handler",
+                    "ImageConfig": "ImageConfig",
+                    "Layers": "Layers",
                     "MemorySize": "MemorySize",
-                    "Layers": "Layers"
+                    "Runtime": "Runtime",
+                    "Role": "Role",
+                    "Timeout": "Timeout",
+                    "TracingConfig": "TracingConfig",
+                    "VpcConfig": "VpcConfig"
                     # TODO add missing fields
```

Comment:
are these complete now? if so we could remove this comment

Reply:
Good catch. Not yet, though - we're still missing a few props like `Tags`, etc. To be done in a future PR 👍